### ISSUE TITLE
Testing cmorization of ec-earth cmip6 NEMO output

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#202]
We are testing now whether `ece2cmor` runs on the ec-earth cmip6 NEMO output, which is produced by using the new NEMO cmip6 `file_def` files as created by using the `generate-ec-earth-namelists.sh` script.

**Comment by treerink** [comment]
First issue seems that in ece2cmor the following ocean frequencies are not implemented yet:
-  odec
- oclim
- ofx

For instance see error message:
`````scripts
ERROR:ece2cmor3.taskloader:Could not find parameter table entry for volcello          in table Odec   ...skipping variable. This variable is ignored
`````

**Comment by treerink** [comment]
Another issue seems related to creating girds:
`````scripts
INFO:ece2cmor3.nemo2cmor:Creating NEMO grids in CMOR...
Traceback (most recent call last):
  File "/usr/people/reerink/anaconda2/envs/ece2cmor3/bin/ece2cmor", line 11, in <module>
    load_entry_point('ece2cmor3==1.0.0', 'console_scripts', 'ece2cmor')()
  File "/usr/people/reerink/anaconda2/envs/ece2cmor3/lib/python2.7/site-packages/ece2cmor3-1.0.0-py2.7.egg/ece2cmor3/ece2cmor.py", line 103, in main
    ece2cmorlib.perform_nemo_tasks(args.datadir, args.exp, startdate, length)
  File "/usr/people/reerink/anaconda2/envs/ece2cmor3/lib/python2.7/site-packages/ece2cmor3-1.0.0-py2.7.egg/ece2cmor3/ece2cmorlib.py", line 177, in perform_nemo_tasks
    nemo2cmor.execute(nemo_tasks)
  File "/usr/people/reerink/anaconda2/envs/ece2cmor3/lib/python2.7/site-packages/ece2cmor3-1.0.0-py2.7.egg/ece2cmor3/nemo2cmor.py", line 67, in execute
    create_grids(tasks)
  File "/usr/people/reerink/anaconda2/envs/ece2cmor3/lib/python2.7/site-packages/ece2cmor3-1.0.0-py2.7.egg/ece2cmor3/nemo2cmor.py", line 284, in create_grids
    grid = read_grid(filename)
  File "/usr/people/reerink/anaconda2/envs/ece2cmor3/lib/python2.7/site-packages/ece2cmor3-1.0.0-py2.7.egg/ece2cmor3/nemo2cmor.py", line 299, in read_grid
    return nemo_grid(name, lons, lats)
  File "/usr/people/reerink/anaconda2/envs/ece2cmor3/lib/python2.7/site-packages/ece2cmor3-1.0.0-py2.7.egg/ece2cmor3/nemo2cmor.py", line 333, in __init__
    self.vertex_lons = nemo_grid.create_vertex_lons(lons_)
  File "/usr/people/reerink/anaconda2/envs/ece2cmor3/lib/python2.7/site-packages/ece2cmor3-1.0.0-py2.7.egg/ece2cmor3/nemo2cmor.py", line 343, in create_vertex_lons
    b[1:nx, 0] = f(0.5 * (a[0:nx - 1, 0] + a[1:nx, 0]))
  File "/usr/people/reerink/anaconda2/envs/ece2cmor3/lib/python2.7/site-packages/numpy/lib/function_base.py", line 2755, in __call__
    return self._vectorize_call(func=func, args=vargs)
  File "/usr/people/reerink/anaconda2/envs/ece2cmor3/lib/python2.7/site-packages/numpy/lib/function_base.py", line 2825, in _vectorize_call
    ufunc, otypes = self._get_ufunc_and_otypes(func=func, args=args)
  File "/usr/people/reerink/anaconda2/envs/ece2cmor3/lib/python2.7/site-packages/numpy/lib/function_base.py", line 2781, in _get_ufunc_and_otypes
    raise ValueError('cannot call `vectorize` on size 0 inputs '
ValueError: cannot call `vectorize` on size 0 inputs unless `otypes` is set
`````

**Comment by aearamos** [comment]
@treerink were you able to solve this issue? I'm having the same issue when trying to cmorize scalar variables. Only after it reads the _scalar files that I get this error and it's probably because of the number of dimensions for scalar grids.

**Comment by aearamos** [comment]
@treerink, what's the status of cmorizing ofx variables for the ocean? I've been trying to run it here but it says it can't find the variables in the nemo output file, such as in
`ERROR:ece2cmor3.nemo2cmor:Variable volcello needed for volcello in table Ofx was not found in NEMO output files... skipping task
`

I have all the variables in my file, but it doesn't seem to work.

**Comment by treerink** [comment]
@aearamos the status is not implemented yet ...

**Comment by treerink** [comment]
In principale CMIP6 output which is based on the automatic generated file_def files can be cmorized now. Some more specific issues take over from this general issue:
ofx variables #249 
odec and oclim #262 
1point file #261

So closing this one.
